charts/langsmith/values.yaml

```diff
@@ -544,7 +548,9 @@ frontend:
       periodSeconds: 10
       timeoutSeconds: 1
     extraContainerConfig: {}
-    extraEnv: []
+    extraEnv:
+      - name: VITE_HOST_ENABLED
+        value: "1"
```

Comment:
`VITE_HOST_ENABLED` needs to be set to `"1"` to enable LangGraph Platform.

TODO: Make this into a templated value.